example/spec/parallel/cnmWorkflow/KinesisTestTriggerSpec.js

```diff
@@ -276,6 +276,14 @@ describe('The Cloud Notification Mechanism Kinesis workflow', () => {
       });
     });
 
+    afterAll(async () => {
+      await removePublishedGranule({
```

Comment:
Two granules are deleted separately, the successful one is published to CMR.
The changes are made so it's easier to see the difference between cnmWorkflow/KinesisTestTriggerSpec.js and KinesisTestTriggerWithUniqueGranuleIdsSpec.js